proto/cosmos/authz/v1beta1/query.proto

```diff
@@ -16,9 +16,14 @@ service Query {
     option (google.api.http).get = "/cosmos/authz/v1beta1/grants";
   }
 
-  // GranterGrants returns list of `Authorization`, granted by granter.
+  // GranterGrants returns list of `GrantAuthorization`, granted by granter.
   rpc GranterGrants(QueryGranterGrantsRequest) returns (QueryGranterGrantsResponse) {
-    option (google.api.http).get = "/cosmos/authz/v1beta1/grants/{granter}";
+    option (google.api.http).get = "/cosmos/authz/v1beta1/grants/granter/{granter}";
+  }
+
+  // GranteeGrants returns a list of `GrantAuthorization` by grantee.
```

Comment:
```suggestion
  // GranteeGrants returns a list of `GrantAuthorization` by grantee.
  //
  // Since: cosmos-sdk 0.46
```

same for all new structs